src/models/candlestickBar.js

```diff
@@ -135,14 +135,17 @@ nv.models.candlestickBar = function() {
                     return open > close ? y(close) - y(open) : y(open) - y(close);
                 });
 
-            container.selectAll('.nv-candlestick-lines').transition()
+            ticks
+                .attr('class', function(d, i, j) { return (getOpen(d, i) > getClose(d, i) ? 'nv-tick negative' : 'nv-tick positive') + ' nv-tick-' + j + '-' + i});
+
```

Comment:
Instead of overwritting the whole 'class' attribute you should use classed:

https://github.com/mbostock/d3/wiki/Selections#classed

Otherwise it's easy to accidentally remove stuff that it shouldn't.